Documentation/ceph-teardown.md

```diff
@@ -119,7 +119,10 @@ The operator is responsible for removing the finalizer after the mounts have bee
 If for some reason the operator is not able to remove the finalizer (ie. the operator is not running anymore), you can delete the finalizer manually with the following command:
 
 ```console
-for CRD in $(kubectl get crd -n rook-ceph | awk '/ceph.rook.io/ {print $1}'); do kubectl patch crd -n rook-ceph $CRD --type merge -p '{"metadata":{"finalizers": [null]}}'; done
+for CRD in $(kubectl get crd -n rook-ceph | awk '/ceph.rook.io/ {print $1}'); do
+    kubectl get -n rook-ceph "$CRD" -o name | \
+   xargs -I {} kubectl patch {} --type merge -p '{"metadata":{"finalizers": [null]}}'
```

Comment:
@leseb Do you want the spaces to be shortened to just 2 and then 4 or?
I added your changes but I'm not quite sure how it should be from your requested changes?

Reply:
I don't have any preference, I'd like to see the same indent that's it :), it looks like it's off by a single character this time :(